Makefile

```diff
@@ -346,7 +346,16 @@ datadir/fewshot: eval/$(release)/train/user.tsv eval/$(release)/dev/user.tsv eva
 	cp eval/$(release)/dev/agent.tsv $@/agent/eval.tsv
 	touch $@
 
-datadir: datadir/agent datadir/nlg datadir/user datadir/fewshot $(all_synthetic_files)
+calibration-ood.tsv:
+	./scripts/make-calibration-ood.sh
+
+datadir/ood: datadir/user calibration-ood.tsv
+	mkdir -p $@
+	cat $(word 2,$^) $</eval.tsv | ./scripts/shuf_with_seed.sh 42 > $@/data.tsv
+	./scripts/split.sh $@/data.tsv $@/train.tsv $@/eval.tsv 0.8
```

Comment:
I just remembered you can use `genie split-train-eval --contextual` here instead, because the datasets are in Genie format. That would be:
```
genie split-train-eval --contextual --split-strategy raw-sentence --eval-probability 0.2 --train $@/train.tsv --eval $@/eval.tsv
```

No need to make the change though, given this code works (and it's a nice trick to use openssl for seeded shuf)